cmd/agent-smoke/main.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"os"
+	"time"
+
+	"github.com/go-kit/log"
+	"github.com/go-kit/log/level"
+	smoke "github.com/grafana/agent/cmd/agent-smoke/internal"
+)
+
+func main() {
+	var (
+		debugLog          bool
```

Comment:
Nit: use `github.com/weaveworks/common/logging.Level` here instead, which implements flag.Value:

```yaml
var (
  logLevel logging.Level 
)

logLevel.RegisterFlags(flag.CommandLine)
```

Reply:
thanks, did this for logging.Format as well :+1: